spec/definitions.yaml

```diff
@@ -89,6 +89,11 @@ definitions:
               default:
                 description: Default number of availability zones for a cluster.
                 type: integer
+              zones:
+                type: array
+                description: The distinct availability zones available in the installation's region.
```

Comment:
Is 'distinct' a necessary word here?

Reply:
Don't know. Calling some native speakers. @rossf7 @JosephSalisbury @tfussell Does it sound better without?